iree/vm/c_funcs.h

```diff
@@ -17,6 +17,13 @@
 
 #include <stdint.h>
 
+// arithmetic
 inline int32_t vm_add_i32(int32_t a, int32_t b) { return a + b; }
 
+// compare
+inline int32_t vm_cmp_ne_i32(int32_t a, int32_t b) { return a != b ? 1 : 0; }
+
+// const
+inline int32_t vm_const_i32(int32_t a) { return a; }
```

Comment:
I don't really get why emitting this function would ever be useful.

Reply:
I see multiple ways of printing constant operations (not having thought about ref types).

1. Convert to calls to "pseudo" functions like it is now
2. Handle this in the printer (CModuleTarget)
3. Convert to std.const operations and let emitc do the printing
4. Like 3 but define an emitc.const op

Reply:
Yeah, I agree that the way this handled now isn't very convenient. However, I would like to avoid to bloat the printer (option 2) and would prefer to handle the const op more MLIR-ish. For MLIR-HLO, I have chosen option 3, see the forked [MHLOToStd](https://github.com/iml130/mlir-emitc/tree/main/lib/Conversion) conversion. So far I don't see strong pros of an emitc.const op (option 4). A conversion might be more straight forward. I would suggest to handle this in a follow up PR.